server/app/auth/oidc/CiviformOidcProfileCreator.java

```diff
@@ -114,16 +114,17 @@ protected CiviFormProfileData mergeCiviFormProfile(
         .forEach(role -> civiformProfile.getProfileData().addRole(role));
     adaptForRole(civiformProfile, roles);
 
+    String emailAddress =
+        getEmail(oidcProfile)
+            .orElseThrow(
```

Comment:
**question (blocking):** Is throwing an exception in this case new behavior?

Reply:
No